aea/connections/local/connection.py

```diff
@@ -138,6 +140,17 @@ def register_service(self, public_key: str, service_description: Description):
         with self._lock:
             self.services[public_key].append(service_description)
 
+    def register_agent(self, public_key: str, agent_description: Description):
```

Comment:
this is not used anywhere - shall we remove it?

Reply:
It is pulled from the feature PR. I messed up with the PRs a little bit.